_maps/map_files/Aegis/aegis3.dmm

```diff
@@ -11,6 +11,19 @@
 /obj/machinery/light/runway,
 /turf/open/space/basic,
 /area/space/nearstation)
+"ad" = (
+/obj/machinery/camera/autoname,
+/obj/machinery/light{
```

Comment:
You can undo your aegis changes and remove it from the basemap and configs to prevent it failing on travis. Its a possible alternative to mapping conflicts which dont need fixing.